### DISCUSSION TITLE
12월 28일 9시 슬라임파티 접속불량

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#148] [Issue Report (Legacy)]
슬라임파티 고장났어요!

**Comment by x4cf** [comment]
당일 10시 30분에 해결됨

**Reply by mhkim0720** [reply]
서버 이상 시  뭔가 공지를 해주 실수는 없는지 문의 하고 싶네요. 이게 개인 개발자는 상관 없는데 회사 일로 하는 사람들은 때때로 곤란한 상황이 연출되기 땜에 ....
